ftw/theming/resources/scss/globals/variables.scss

```diff
@@ -135,144 +173,7 @@ $spinner-url: "" !default;
   font-weight-heading,
   max-width-page,
   line-height-base,
-  spinner-url);
-
-/* Viewports */
-
-// Taken from http://en.wikipedia.org/wiki/Display_resolution
-$desktop-WQHD-width:   2560px !default;
-$desktop-FHD-width:    1920px !default;
-$desktop-HDPLUS-width: 1600px !default;
-$desktop-HD-width:     1360px !default;
-$desktop-WXGA-width:   1280px !default;
-$desktop-XGA-width:    1024px !default;
-
-// Taken from http://www.ciop.com/common-resolutions-for-mobile-phones-and-tablets
-$mobile-large: 1280px !default;
-$mobile-medium: 1024px !default;
-$mobile-small: 800px !default;
-$mobile-smaller: 600px !default;
-$mobile-tiny: 480px !default;
-
-$screen-XL: "(min-width: #{$desktop-WQHD-width})" !default;
-$screen-L: "(min-width: #{$desktop-HDPLUS-width})" !default;
-$screen-M: "(min-width: #{$desktop-HD-width})" !default;
-$screen-S: "(min-width: #{$desktop-XGA-width})" !default;
-$screen-XS: "(min-width: #{$mobile-small})" !default;
```

Comment:
Can you show me where these variables are used. I could not find any dependencies on these variables.